docs/features/kubernetes/configuration.md

```diff
@@ -94,7 +95,12 @@ The base URL to the Kubernetes control plane. Can be found by using the
 ##### `clusters.\*.name`
 
 A name to represent this cluster, this must be unique within the `clusters`
-array. Users will see this value in the Software Catalog Kubernetes plugin.
+array. Users will see this value in the Software Catalog Kubernetes plugin if `clusters.\*.displayName` is not set.
```

Comment:
you might need a different line break here. Might fail validation 🤔 
I would probably put each sentence in a separate line

Reply:
done